tests/lib/rules/no-tryinvoke.js

```diff
@@ -0,0 +1,40 @@
+const rule = require('../../../lib/rules/no-tryinvoke');
+const RuleTester = require('eslint').RuleTester;
+
+const { ERROR_MESSAGE } = rule;
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('no-tryinvoke', rule, {
+  valid: ['this.foo?.()'],
+  invalid: [
+    {
+      code: `
+        import { tryInvoke } from '@ember/utils';
+        tryInvoke(this, 'foo');
+      `,
+      output: null,
+      errors: [
+        {
+          message: ERROR_MESSAGE,
```

Comment:
Can you add `type` to the error objects? That asserts what type of node the violation is flagged on.